src/app.js

```diff
@@ -1,21 +1,39 @@
+/* eslint-disable no-console */
+
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const PORT = 3001;
+
+const server = http.createServer((req, resp) => {
+  resp.setHeader('Content-Type', 'application/json');
```

Comment:
This header is wrong, we are not sending json here at all.

Use `application/octet-stream` for random files, and `text/plain` for your error message about `/file`.

[Docs about this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)